EIPS/eip-1238.md

```diff
@@ -0,0 +1,231 @@
+---
+eip: 1238
+title: Non-transferable Tokens
+description: An interface for fungible and non-fungible, non-transferable tokens.
+author: Raphael Roullet (@ra-phael), Chris Chung (@0xpApaSmURf), Nicola Greco (@nicola)
+discussions-to: https://ethereum-magicians.org/t/eip-1238-non-transferable-tokens/9044
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-09-07
+requires: 165, 712
+---
+
+## Abstract
+
+A _badge_ or non-transferable token (_NTT_) is a token that cannot be transferred once assigned. Badges can be accumulated through time and put at stake. Simply speaking, badges are statements about a public key: they can be quantitative (e.g. reputation, experience) or qualitative (badges, titles).
```

Comment:
> Badges can be accumulated through time and put at stake.

I am not sure what "put at stake" means here. Is this some kind of staking mechanism, or are you talking about the badge issuer's reputation, or something else entirely?